resources/data/darwin_deps-0.9.3.json

```diff
@@ -78,6 +78,43 @@
                 }
             ]
         },
+        {
+            "type": "Checkable",
+            "steps": [
+                {
+                    "type": "Step",
+                    "title": "Install GN build system",
+                    "description": [
+                        "_Note: If you are not interested in the [Connected Home over IP](https://www.connectedhomeip.com/) applications, you can omit steps presented in this section._",
+                        "Download and extract [GN](https://gn.googlesource.com/gn/) build system executable archive used by the Connected Home over IP to generate output files:",
```

Comment:
Shouldn't this be a proper step, introduced with a number?

Reply:
Done